pkg/registries/registries_test.go

```diff
@@ -219,7 +226,261 @@ func TestMergedMirrorSets(t *testing.T) {
 					},
 				})
 			}
-			res, err := mergedMirrorSets(in)
+			res, err := mergedICSPMirrorSets(in)
+			if err != nil {
+				t.Errorf("Error %v", err)
+				return
+			}
+			assert.Equal(t, c.result, res)
+		})
+	}
+}
+
+func TestMergedTagMirrorSets(t *testing.T) {
```

Comment:
If the tests are copy&pasted, could they instead be moved into a single global variable, with the tests inner loops making any type conversions that are necessary?

(All of this might get simpler if most of `the mergedMirrorSets` core logic can be shared.)

Reply:
fixed